README.md

```diff
@@ -30,6 +30,7 @@ Use file name based on RFE id, for example QUARKUS-123.md
  - Focus on exploratory testing of the feature
  - Ensure good user experience and simplicity of use
  - Check impact on disk + memory footprint and boot time
+ - Check the community status of the extension/feature, preview and experimental ones should be considered carefully
```

Comment:
1. Preview extensions are marked as such in code.quarkus. Is there something similar for features?
2. I would like to have more context on "should be considered carefully". Can we decide not to cover the extension if it is in preview status? Or it is more like "make you coverage ready to sudden changes in upstream"?

Reply:
@rsvoboda organize it as you see fit, but people, who write test plans, should know about the context you described earlier.

Reply:
> @rsvoboda organize it as you see fit, but people, who write test plans, should know about the context you described earlier.

+1

Reply:
Updated